### ISSUE TITLE
Use thumbnail (Only audio mode) 

### ISSUE THREAD

**Issue by wuniversales** [futo-org/grayjay-android#1525]
### Use Cases

When playing only music, instead of showing a black screen, show the video thumbnail?

**Comment by nastynick64** [comment]
![Image](https://github.com/user-attachments/assets/21b5b657-35fb-43b3-8129-fd457da35b06)

It looks like this was implemented at some point.

**Comment by wuniversales** [comment]
Not entirely. If you change songs while in audio mode, the screen goes black again.

**Comment by nastynick64** [comment]
> Not entirely. If you change songs while in audio mode, the screen goes black again.

True, I just noticed this too. If you press on the Background button the current video plays on the background and the thumbnail shows up, but if you close out of the video and open a new video it plays in the background, but DOES NOT show the thumbnail, just a black screen.

**Comment by Zvonimir-FUTO** [comment]
> Not entirely. If you change songs while in audio mode, the screen goes black again.

Could you please open a bug report for this?
